### PR TITLE
feat(ui): 2. deprecated accounts warning in account list

### PR DESCRIPTION
**Pull request by delaaxe**

### CODE REVIEW

**Review by dhruvkelawala — APPROVED** [review]

File: packages/extension/src/shared/wallet.service.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+// from https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2645.md
+// m / purpose' / layer' / application' / eth_address_1' / eth_address_2' / index
+
+export const baseDerivationPath = "m/44'/9004'/0'/0"
+
+interface WithSigner {
+  signer: WalletAccountSigner
+}
+
+export const hasLatestDerivationPath = ({ signer }: WithSigner) =>
```
Comment (janek26) [inline packages/extension/src/shared/wallet.service.ts:12]: should we return boolean explicitly and wrap the implementation in `Boolean()` ?

File: packages/extension/src/ui/features/accounts/AccountListScreen.tsx [hunk 2]
```diff
@@ -54,6 +57,10 @@ export const AccountListScreen: FC = () => {
   const { isBackupRequired } = useBackupRequired()
 
   const accountsList = Object.values(accounts)
+  const [newAccounts, deprecatedAccounts] = partition(
```
Comment (janek26) [inline packages/extension/src/ui/features/accounts/AccountListScreen.tsx:60]: really enjoyed this code

**Review by janek26 — APPROVED** [review]

File: packages/extension/src/ui/features/accounts/AccountListScreen.tsx (hunk 2 above)
Comment (dhruvkelawala) [reply packages/extension/src/ui/features/accounts/AccountListScreen.tsx:60]: Me too. Learnt something new. xD

File: packages/extension/src/shared/wallet.service.ts (hunk 1 above)
Comment (delaaxe) [reply packages/extension/src/shared/wallet.service.ts:12]: I think optional Boolean is always fine

**Pull request merged**